doc/en/user/source/services/wfs/webadmin.rst

```diff
@@ -66,6 +66,65 @@ A list of values to be declared in all feature types can be provided in the WFS
 
 The list will be used only for the capabilities document generation. It does not limit the actual target SRS usage in GetFeature requests.
 
+
+CSV
+---
+.. figure:: img/services_WFS_csv.png
+
+   WFS configuration options - CSV section 
+
+CSV is still widely used as a format to exchange tabular data. For GeoServer CSV output format, the date field format can specified using the Date Format text box as shown above.
+Here are some common formatting patterns supported
+
+.. list-table::
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Date and Time Pattern 
+     - Result
+   * - ``yyyy.MM.dd G 'at' HH:mm:ss z``
+     - 2001.07.04 AD at 12:08:56 PDT  
+   * - ``EEE, MMM d, ''yy``
+     - Wed, Jul 4, '01
+   * - ``yyyy-MM-dd'T'HH:mm:ss.SSSZ``
+     - 2001-07-04T12:08:56.235-0700 
+   * - ``yyyy-MM-dd'T'HH:mm:ss.SSSXXX``
+     - 2001-07-04T12:08:56.235-07:00
+
+
+Here, **yyyy-MM-dd'T'HH:mm:ss.SSS'Z'** pattern represents the year, month, day, hour, minute, second and milliseconds. The components are separated by a hyphen character. A literal 'T' character is used to separate the date and time parts. A literal 'Z' character represents the UTC time zone. For instance, yyyy represents the year with four digits, MM represents the month with leading zeros, dd represents the day with leading zeros, and so on.
+
+Similarly, patterns can be formed by using the characters provided below
```

Comment:
The `SSS` pattern is used above, but not shown below. It might be worth including that, since it seems like a pretty common need.